testData/preferences/samuel.json5

```diff
@@ -0,0 +1,81 @@
+// # Samuel.json5
+//
+// This preference set changes several preferences from ZoomText 2019
+//
+// ## Testing
+//
+// This changes the following settings from the AT software:
+//
+//      - Voice Language: Spanish
+//      - Synthesizer: Vocalizer Expressive
+//      - Voice Settings:
+//          + Rate: 120 WPM.
+//          + Volume: 70%.
+//          + Pitch: 35%.
+//
+//      - Color Scheme: Black and White
+//
+//      - Keyboard Echo: Words
+//      - Mouse Echo:
+//          + Hover: Enabled
+//          + Echo the whole line: Enabled
+//
+//      - Pointer Scheme: Giant Green
+//      - Zoom Level: 1.4
+//      - Navigation Settings - Mouse - "Router Pointer into view when it is moved": Enabled.
+//      - Navigation Settings - Mouse - "Route pointer over the active control": Enabled
+//      - Navigation Settings - Mouse - "Hold Shit key to move only horizontally or vertically": Enabled
+//      - Navigation Settings - Mouse - "Hold Ctrl key to stay within the active window": Enabled
+{
+    "flat": {
+        "name": "Samuel",
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/applications/com.aisquared.zoomtext": {
+                        "ZtSpeech.SapiProductName": "VocalizerExpressive-Spanish",
+                        "ZtSpeech.Language": 10,
+
+                        "ColorScheme.ColorSchEnable": 1,
+                        "ColorScheme.ColorSchIndex": 5,
+
+                        // [VocalizerExpressive-Spanish Speech - Screen Reader]
+                        // ==================================================================
+                        "VocalizerExpressive-Spanish Speech - Screen Reader.Speaker": "",
```

Comment:
These keys are insane.  We should never let anyone store these, and should use user-facing aliases instead.

Reply:
probably, I guess I'll just leave them for now, similarly to how we did with the JAWS local language ones.

Reply:
I'll have to work on these when I formalise how we handle "intra-application" settings.